### PR TITLE
typo: privacty -> privacy

### PR DESCRIPTION
**Pull request by nschonni**
Trim trailing/leading spaces

### CODE REVIEW

File: docs/framework/configure-apps/file-schema/wcf/ws2007federationhttpbinding.md [hunk 1]
```diff
@@ -3,16 +3,18 @@ title: "<ws2007FederationHttpBinding>"
 ms.date: "03/30/2017"
 ms.assetid: 9af4ec79-cdef-457e-9dca-09d5eb821594
 ---
+
```
Comment (NextTurn) [inline docs/framework/configure-apps/file-schema/wcf/ws2007federationhttpbinding.md]: Docs Content Validation Warning: Markdown content is not allowed before H1.

BTW the blank lines after metadata headers are being removed in MicrosoftDocs/PowerShell-Docs repo. Any consensus?

Comment (nschonni) [reply docs/framework/configure-apps/file-schema/wcf/ws2007federationhttpbinding.md]: @meganbradley should a blank line between the front matter be triggering the rule? The MD041 doesn't https://github.com/DavidAnson/markdownlint/blob/master/doc/Rules.md#md041

Comment (nschonni) [reply docs/framework/configure-apps/file-schema/wcf/ws2007federationhttpbinding.md]: Nevermind, I realized this is a file with a BOM, which is triggering it. That might be a different issue that @mairaw said she's already flagged

Comment (NextTurn) [reply docs/framework/configure-apps/file-schema/wcf/ws2007federationhttpbinding.md]: I don't think BOM is needed anywhere in this repo. Can we open a PR to remove them all?

Comment (nschonni) [reply docs/framework/configure-apps/file-schema/wcf/ws2007federationhttpbinding.md]: I'm running a script right now to strip them from the markdown files. I'll see how many are impacted and PR it

Comment (nschonni) [reply docs/framework/configure-apps/file-schema/wcf/ws2007federationhttpbinding.md]: Opened #10963

**Review by BillWagner — APPROVED** [review]
LGTM. :shipit:

**Pull request merged**